public_html/index-v1.php

```diff
@@ -1,330 +0,0 @@
-<?php
-// Get absolute path to find our includes files
-// NOTE - we can't get our @var $base-url until we call functions
-global $base_dir;
-$base_dir = rtrim( dirname( realpath( __FILE__ ) ), '/' );
```

Comment:
v1 index is now deleted. and, in truth, it'll be on github if we ever want to look at it.